src/renderGenerator.js

```diff
@@ -127,7 +127,7 @@ module.exports = {
     },
 
     errorPageHandler: function(req, res, renderdata, next){
-        var user = req.session.user || {},
+        var user = req.session?.user || {},
```

Comment:
If the error page is being shown as a result of a deeper connection issue, `session` doesnt exist and the error page itself throws an additional error.